proto/ord_schema.proto

```diff
@@ -186,74 +284,21 @@ message Amount {
   }
   // Precision of the measurement, as a percentage of the value.
   float precision_in_percent = 5;
-  // TODO(ccoley): fix this bastardization of proto and json
-  /**
-   * For pre-mixed solutions when value given in concentration, the solvent
-   * is specified as an additional Compound, which should have an identifier
-   * and an amount specified by volume. For example, suppose we are adding
-   * 3 mL of a 4 M solution of NaOH in water.
-   * 
-   * compound {
-   *   identifiers: [
-   *     {type: IDENTIFIER_SMILES, value: "[Na+].[OH-]"},
-   *     {type: IDENTIFIER_NAME, value: "sodium hydroxide"}
-   *   ]
-   *   amount {
-   *     volume: {value: 3, units: VOLUME_UNITS_MILLILITER}
-   *     concentration: {value: 4, units: CONCENTRATION_UNITS_MOLAR}
-   *     solvent {
-   *       identifiers: [
-   *         {type: IDENTIFIER_SMILES, value: "O"},
-   *         {type: IDENTIFIER_NAME, value: "water"}
-   *       ]
-   *     }
-   *   }
-   * }
-   */
-  // The concentration of the pre-mixed solution as-added, *not* the final
-  // concentration in the overall crude reaction/product mixture.
-  Concentration concentration = 6;
-  repeated Compound solvent = 7;
-
-  // TODO(ccoley) Also fix this one
-  /**
-   * For catalysts on supports. For example, a reaction using 5 grams of
-   * 10% Pd/C might be reported as...
-   * 
-   * compound {
-   *   identifiers: [
-   *     {type: IDENTIFIER_SMILES, value: "[Pd]"},
-   *     {type: IDENTIFIER_NAME, value: "palladium"}
-   *   ]
-   *   amount {
-   *     mass: {value: 5, units: MASS_UNITS_GRAM}
-   *     support {
-   *       identifiers: [
-   *         {type: IDENTIFIER_NAME, value: "carbon"}
-   *       ]
-   *     }
-   *     weight_percent: 10
-   *   }
-   * }
-   *
-   * Immobilized proteins should also use the "support" field.
-   */
-  Compound support = 8;
-  float weight_percent = 9;
 }
 
 message ReactionSetup {
   enum VesselType {
     VESSEL_TYPE_UNKNOWN = 0;
     VESSEL_TYPE_ROUND_BOTTOM_FLASK = 1;
-    VESSEL_TYPE_WELL_PLATE = 2;
-    VESSEL_TYPE_MICROWAVE_VIAL = 3;
-    VESSEL_TYPE_TUBE = 4;
-    VESSEL_TYPE_CONTINUOUS_STIRRED_TANK_REACTOR = 5;
-    VESSEL_TYPE_PACKED_BED_REACTOR = 6;
+    VESSEL_TYPE_VIAL = 2;  // Regular glass vial 
```

Comment:
Useful to distinguish between glass/plastic?

Reply:
Yes, good point. Would a separate enum be overkill? Not all plastics are equal, but I do think that ```VESSEL_MATERIAL_xxxx``` could still be useful